AMS Journals.js

```diff
@@ -9,19 +9,19 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcsibv",
-	"lastUpdated": "2013-12-04 16:57:35"
+	"lastUpdated": "2016-09-03 11:08:35"
 }
 
 function detectWeb(doc, url) {
 	if (url.match(/home\.html|\d{4}[^\/]*\/.+/)) {
 		return "journalArticle";
-	}
-	/*multiples are currently broken
-	else if (url.match(/jour(nals|search)/)) {
+	} else if (url.match(/jour(nals|search)/)) {
```

Comment:
The same name is used several lines below in the `doWeb`. However, currently the search inside a journal directs to another website and this service is not available at the moment. I will rewrite this part to use also `getSearchResults`-function and then we can check on that. Give me just some minutes...